parser-typechecker/src/Unison/Runtime/IR.hs

```diff
@@ -1027,7 +1033,8 @@ constructorId v = case v of
   Requested _ -> 10
   Ref{} -> 11
   Cont _ -> 12
-  UninitializedLetRecSlot{} -> 13
+  C _ -> 13
+  UninitializedLetRecSlot{} -> 14
```

Comment:
While it might be okay in this case, in general, we don't wanna _change_ these numbers. (13 -> 14)

Reply:
I missed this one. In this case it’s fine since we aren’t persisting runtime values yet anywhere but yeah in general don’t change any of the tag values.